cinema/models.py

```diff
@@ -1,10 +1,36 @@
 from django.db import models
 
 
+class Actor(models.Model):
+    first_name = models.CharField(max_length=63)
+    last_name = models.CharField(max_length=63)
+
+    def __str__(self):
+        return f"{self.first_name} {self.last_name}"
+
+
+class Genre(models.Model):
+    name = models.CharField(max_length=63, unique=True)
+
+    def __str__(self):
+        return self.name
+
+
+class CinemaHall(models.Model):
+    name = models.CharField(max_length=63)
+    rows = models.IntegerField()
+    seats_in_row = models.IntegerField()
```

Comment:
I think we should check if someone passes a negative value, just for stability in our db